app/platform/fabric/Proxy.ts

```diff
@@ -391,4 +395,207 @@ export class Proxy {
 			);
 		}
 	}
+
+	/**
+	 *
+	 *
+	 * @param {*} channel_genesis_hash
+	 * @param {*} blockNo
+	 * @returns
+	 * @memberof Proxy
+	 */
+	async fetchDataByBlockNo(network_id, channel_genesis_hash, blockNo) {
+		const blockValue = await this.dataByBlockNo(network_id, channel_genesis_hash, blockNo);
+		if (blockValue) {
+			return blockValue;
+		}
+		else
+			return `No Block found with this block no # ${blockNo}`;
```

Comment:
Please follow Principle of least astonishment.

Reply:
@adityajoshi12 Is it fine to change the return message as 'No block found' ?